plugins/org.obeonetwork.m2doc/src/org/obeonetwork/m2doc/provider/ProviderConstants.java

```diff
@@ -38,6 +38,10 @@
      */
     String DIAGRAM_ACTIVATED_LAYERS_KEY = "layers";
     /**
+     * The key used in the map passed to {@link IProvider} to define a flag indicate if representation (Sirius diagram or table) should be refresh before M2Doc generation.
+     */
+    String REPRESENTATION_FLAG_REFRESH_KEY = "refreshFlag";
```

Comment:
Should be called 'refreshRepresentation' shouldn't it? Why is this name different from the option name?